javascript/javascript_in_the_real_world/linting.md

```diff
@@ -52,6 +52,6 @@ Another way to address the conflict is to use `eslint-plugin-prettier`. It lets
 
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">What is linting?</a>
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">Which problems can linting prevent?</a>
-- <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=hkfBvpEfWdA">Why should you use Prettier?</a>
+- [What is linting](https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/)
```

Comment:
This is looking great [on the preview](https://www.theodinproject.com/lessons/preview?uuid=138a5ff5-93f2-47ba-9cf9-7c2610dbb443), but theres just one small typo. We need a question mark here:

```suggestion
- [What is linting?](https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/)
```

Reply:
Done! So excited. This is my first time contributing to open source.